.mergify.yml

```diff
@@ -2,7 +2,10 @@ pull_request_rules:
   - name: comment on broken dependabot PRs, asking for help
     conditions:
       - "author=dependabot-preview[bot]"
-      - "status-failure=ci/circleci: debug-build-test"
+      - "status-failure=static_analysis"
```

Comment:
This whole rule doesn't really make sense because it never worked the way it was intended (mergify cannot assign for some reason).

I think these conditions are combined with a logical AND so now that it is more than one, I think it makes no more sense. It is unlikely a dependency upgrade breaks all of those.

Reply:
Ooooh. Cool, I can update to something that makes sense.